deployment/README.md

```diff
@@ -20,6 +20,18 @@ the project:
 gcloud auth application-default login
 ```
 
+Then, specify your Google Cloud project ID:
+```bash
+export GOOGLE_PROJECT={VALUE}
```

Comment:
Could use this:
```bash
export GOOGLE_PROJECT=$(gcloud config get-value project 2> /dev/null)
```

Reply:
Maybe that works if you run this command after you've ran `gcloud auth` from a VM inside a project, but that might not always be the case. Otherwise, how can gcloud know which project it should select? When you authenticate via a web link, it doesn't give you the option to select the project.

Reply:
Sadness :(